.github/ISSUE_TEMPLATE/02_api_proposal.yml

```diff
@@ -33,15 +33,15 @@ body:
                 public void Fancy(T item);
             }
         }
-        ```     
+        ```
     validations:
       required: true
   - type: textarea
     id: api-usage
     attributes:
       label: API Usage
       description: |
-        Please provide code examples that highlight how the proposed API additions are meant to be consumed. This will help suggest whether the API has the right shape to be functional, performant and useable.
+        Please provide code examples that highlight how the proposed API additions are meant to be consumed. This will help suggest whether the API has the right shape to be functional, performant and usable.
```

Comment:
useable is less frequently used spelling, but it is not wrong.

https://grammarhow.com/useable-or-usable/

Reply:
I just compared their frequency repo-wide and streamlined on `usable`:

```sh
> git grep -i usable | wc -l
421

> git grep -i useable | wc -l
19
```